docs/CONTRIBUTING.md

```diff
@@ -38,12 +48,79 @@ template will be requested to be updated. Any pull requests opened without a
 corresponding issue may be delayed or be required to undergo further changes
 before being accepted.
 
-### Issues
+#### Pull Request Workflow
```

Comment:
I don't think this belongs in the repo.

If anything, there should just be a simple link to the existing GitHub docs such as: 
* https://help.github.com/en/github/collaborating-with-issues-and-pull-requests

Reply:
The link leads to something a lot more complicated than what I wrote. But it is indeed not what the repo is about. So I'll replace the description with the link.